README.md

```diff
@@ -1,70 +1,69 @@
-# Getting Started with Create React App
+# redux-bookstore
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-## Available Scripts
+> The Bookstore website displays a list of books, allows one to add a book and remove a selected book.
 
-In the project directory, you can run:
+## Built With
 
-### `npm start`
+- React
+- Redux
+- Css
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## Installation
+Go to your terminal, navigate to your working directory and run
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+`git clone https://github.com/Buyaki01/redux-bookstore`
 
-### `npm test`
+After that navigate to the newly created folder
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+`cd redux-bookstore`
 
-### `npm run build`
+You now have access to the files.
+To install necessary dependencies run
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+`npm install`
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+Start a local server running the command
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+`npm start`
 
-### `npm run eject`
+You should now have the project running locally on a dev server.
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+## Check linter errors:
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+- Run: npx eslint .
+- For css run: npx stylelint "**/*.{css,scss}"
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+## Author 1
```

Comment:
- [ ] OPTIONAL
Kindly change this title `Author 1` to Author` so as to make your ReadMe appear more professional